From a3b35cf8638ab6be66384c96de9811a8fe2e9322 Mon Sep 17 00:00:00 2001
From: Yanhan0507 <yhzhang@utexas.edu>
Date: Tue, 22 Sep 2015 15:13:04 -0500
Subject: [PATCH] fourth1

---
 HelloWorld/src/HelloWorld.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/HelloWorld/src/HelloWorld.java b/HelloWorld/src/HelloWorld.java
index 8785566..3a4da99 100644
--- a/HelloWorld/src/HelloWorld.java
+++ b/HelloWorld/src/HelloWorld.java
@@ -1,6 +1,6 @@
 public class HelloWorld {
 	public static void main( String [] args){
-		System.out.println("Boujour, monde");
+		System.out.println("Hello, World");
 	}
 
 }
-- 
1.9.5 (Apple Git-50.3)

